immunopepper/utils.py

```diff
@@ -167,7 +166,7 @@ def _get_variant_pos_offset(variant_pos, coord_pair_list, strand):
         offset = 0
         takes_effect = False
         for pair in coord_pair_list:
```

Comment:
you could do `for a, b in coord_pair_list` or `for a, b, *_ in coord_pair_list` depending how the sublists look like

Reply:
good point